src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -13,6 +16,13 @@ import scala.util.Try
 
 class LocalBackend extends Backend {
 
+  implicit class WriteWithNewline(writer: Writer) {
```

Comment:
Right but now you're getting a new one for every instance of LocalBackend, if you put it in the companion object there's only one of them